.github/workflows/deploy-static-site.yml

```
@@ -0,0 +1,44 @@
+# This workflow will build the source code, authenticate with Azure and then upload the built application to a storage account
+
+name: Deploy Static Site
+
+on:
+  workflow_run:
+    workflows: ["Node.js CI"]
+    types: [completed]
+    branches: [main, add-deployment-pipeline]
```

```suggestion
    branches: [main]
```
Don't merge with this branch name in the list.